src/schema/slack/functions/_scripts/src/templates/test/template_utils_test.ts

```diff
@@ -0,0 +1,94 @@
+export { assertEquals } from "https://deno.land/std@0.152.0/testing/asserts.ts";
+import {
+  autogeneratedComment,
+  getFunctionName,
+  getSlackCallbackId,
+  renderFunctionImport,
+  renderTypeImports,
+} from "../template_utils.ts";
+import {
+  assertEquals,
+  assertStringIncludes,
+} from "https://deno.land/std@0.152.0/testing/asserts.ts";
```

Comment:
The [Deno standard for managing dependencies](https://deno.land/manual@v1.30.3/examples/manage_dependencies) is to use a `dev_deps` file to manage importing from external libraries for dev only dependencies